main.go

```diff
@@ -127,7 +129,16 @@ func main() {
 	if flagsSet["kube-api-burst"] {
 		kubeConfig.Burst = burst
 	}
-
+	// Metrics endpoint is enabled in 'config/default/kustomization.yaml'. The Metrics options configure the server.
+	// More info:
+	// - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.19.1/pkg/metrics/server
+	// - https://book.kubebuilder.io/reference/metrics.html
+	metricsServerOptions := metricsserver.Options{
+		BindAddress:    metricsAddr,
+		SecureServing:  true,
+		FilterProvider: filters.WithAuthenticationAndAuthorization,
```

Comment:
@kannon92 it seems that we should disable http/2: https://github.com/kubernetes-sigs/lws/pull/284#discussion_r1888126368